lib/widgets/breez_avatar_dialog.dart

```diff
@@ -195,35 +188,25 @@ Widget breezAvatarDialog(BuildContext context, UserProfileBloc userBloc) {
               child: Text('SAVE',
                   style: Theme.of(context).primaryTextTheme.button),
               onPressed: () async {
```

Comment:
The button should be disabled during upload or when there is no profile image chosen.
```suggestion
              onPressed: _pickedImage == null || _isUploading ? null : () async {
```

Reply:
It should be enabled if there's no profile image chosen because the user could only be changing their name or generated an avatar using "Random" button.

Reply:
OK, then in the case there is no _pickedImage just don't call to uploadImage.